cast/Cargo.toml

```diff
@@ -31,6 +31,7 @@ hex = "0.4.3"
 # aws
 rusoto_core = { version = "0.48.0", default-features = false, optional = true }
 rusoto_kms = { version = "0.48.0", default-features = false, optional = true }
+evm-disassembler = "0.1.0"
```

Comment:
Ups, had forgotten to make the repo public. Should be fixed now. Thanks for letting me know.

Reply:
I did not know about heimdall actually. I does include disassembly as well, the result of which is pretty much the same as in my crate. (with slightly different formatting).
It also includes a bunch of other features (decompiling to solidity etc. ). 

In short the crate I wrote is a more "lightweight" option limited to the disassembly feature alone.

If we want to support other heimdall features it might be worth switching over, which should be pretty straight forward.

Reply:
A few nits on your crate which uses unnecessary owned values instead of references:

- `disassemble_str(&str) -> _ { ... hex::decode(&input) ...} `, no need for .to_owned(), I think this can be caught with running `clippy`?
- `disassemble_bytes(- Vec<u8> + &[u8]) -> _`, and substitute `VecDeque::from .. .pop_front()` with `bytes.iter() .. .next()`
- `format_operations(- Vec<Operation> + &[Operation]) -> String { ... for operation in operations.iter() { string.write_fmt("{operation:?}\n"); } ... }`
- `decode_operation(&mut impl Iterator<Item = u8>)`, same as in `disassemble_bytes`

Otherwise this is OK. This can maybe also moved to a separate cast subcommand, like cast disassemble/dasm/opcode

Reply:
@DaniPopes Thanks so much for taking the time and the detailed feedback. I'll work in those suggestions into the crate🙏

Interestingly the [clippy step in my CI](https://github.com/ckoopmann/evm-disassembler/actions/runs/4382892795/jobs/7672476418) did not catch those, but maybe It's because I'm not using the nightly version. 🤔

Reply:
I applied the suggested changes to the crate [here](https://github.com/ckoopmann/evm-disassembler/pull/7), and bumped the version.

Reply:
As suggested, I also added `disassemble` as an individual cast command (https://github.com/foundry-rs/foundry/pull/4518/commits/060d0c89f8ad3a73401b7a6725fec89112c1da05):
<img width="607" alt="image" src="https://user-images.githubusercontent.com/15629702/224626564-22d8efa6-ba90-44b4-9467-788fc06c5ca5.png">